src/App.js

```diff
@@ -1,7 +1,16 @@
 import React from "react";
+import Friend from "./components/fetchedData";
```

Comment:
why is this file called `fetchData` ?
Also since it returning component use CapitalCase